imagebuildah/build.go

```diff
@@ -483,6 +483,19 @@ func (s *StageExecutor) volumeCacheRestore() error {
 // imagebuilder tells us the instruction was "ADD" and not "COPY".
 func (s *StageExecutor) Copy(excludes []string, copies ...imagebuilder.Copy) error {
 	for _, copy := range copies {
+		// If the file exists, check to see if it's a symlink.
+		// If it is a symlink, convert to it's target otherwise
+		// the symlink will be overwritten.
+		fileDest, _ := os.Stat(s.mountPoint + copy.Dest)
+		if fileDest != nil {
+			if !(fileDest.Mode()&os.ModeSymlink == os.ModeSymlink) {
```

Comment:
Did you really mean `!` ?

Reply:
It seemed to work OK in my testing, but your suggestion is shorter and seems to work just fine, converted.

Reply:
In UNIX-land it's important to mask then check for `S_IFLNK` because that has two bits set. In Go-land `os.ModeSymlink` is a single bit, so (assuming that never changes) it's safe to `!= 0`. I would welcome feedback from someone more Go-literate, though. (And the `!` definitely needed to go)